File: tools/activate-workflow.py
import sqlite3
import sys

# Connect to n8n database
db_path = '/home/node/.n8n/database.sqlite'
conn = sqlite3.connect(db_path)
cursor = conn.cursor()

# Find workflow
cursor.execute("SELECT id, name, active FROM workflow_entity WHERE name LIKE '%Observer%'")
workflow = cursor.fetchone()

if workflow:
    wf_id, wf_name, is_active = workflow
    print(f"Found workflow: {wf_name} (ID: {wf_id}, Active: {is_active})")
    
    if is_active == 0:
        # Activate it
        cursor.execute("UPDATE workflow_entity SET active = 1 WHERE id = ?", (wf_id,))
        conn.commit()
        print("✅ Workflow activated successfully!")
    else:
        print("✅ Workflow is already active!")
else:
    print("❌ Workflow not found")
    sys.exit(1)

conn.close()

File: tools/activate-workflow.js
// Activate workflow in n8n database
const sqlite3 = require('better-sqlite3');
const db = sqlite3('/home/node/.n8n/database.sqlite');

// Find workflow
const workflow = db.prepare("SELECT id, name, active FROM workflow_entity WHERE name LIKE '%Observer%'").get();

if (workflow) {
    console.log(`Found workflow: ${workflow.name} (ID: ${workflow.id}, Active: ${workflow.active})`);
    
    if (workflow.active === 0) {
        // Activate it
        db.prepare("UPDATE workflow_entity SET active = 1 WHERE id = ?").run(workflow.id);
        console.log('✅ Workflow activated successfully!');
    } else {
        console.log('✅ Workflow is already active!');
    }
} else {
    console.log('❌ Workflow not found');
}

db.close();
